src/boot/efi/stub.c

```diff
@@ -35,7 +35,9 @@ enum {
         INITRD_CREDENTIAL = _INITRD_DYNAMIC_FIRST,
         INITRD_GLOBAL_CREDENTIAL,
         INITRD_CONFEXT,
+        INITRD_VENDOR_CONFEXT,
```

Comment:
i can see the point of vendor sysext. but vendor confext, does that make sense? I am not convinced. the separation of /etc/ and /usr/ is too a large degree one of ownership (i.e. etc = user/admin; usr = os/vendor). hence it doesn#t seem right if device vendor can play games in /etc/.